### ISSUE TITLE
Config 2 failed with 'Error: [object Object]'

### ISSUE THREAD

**Issue by numbqq** [rufuspollock/s3-bucket-listing#80]
I try to use config `2. Use Amazon S3 in website mode with URL navigation` but failed with error `Error: [object Object]`.

Bucket: `dl.khadas.com`

My `index.html` is:
```
<!DOCTYPE html>
<html>
<head>
  <title>S3 Bucket Listing Generator</title>
</head>
<body>
  <div id="navigation"></div>
  <div id="listing"></div>

<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
<script type="text/javascript">
 var S3BL_IGNORE_PATH = false;
 var BUCKET_URL = 'https://dl.khadas.com.s3-us-east-1.amazonaws.com';
</script>
<script type="text/javascript" src="https://rawgit.com/rufuspollock/s3-bucket-listing/gh-pages/list.js"></script>
</body>
</html>
```

Bucket : `Everyone` have the `List` and `View` permissions.

Policies:
```
{
    "Version": "2008-10-17",
    "Statement": [
        {
            "Sid": "AllowPublicRead",
            "Effect": "Allow",
            "Principal": {
                "AWS": "*"
            },
            "Action": "s3:GetObject",
            "Resource": "arn:aws:s3:::dl.khadas.com/*"
        }
    ]
}
```

CORS policy:
```
<?xml version="1.0" encoding="UTF-8"?>
<CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
<CORSRule>
    <AllowedOrigin>*</AllowedOrigin>
    <AllowedMethod>GET</AllowedMethod>
    <AllowedHeader>*</AllowedHeader>
</CORSRule>
</CORSConfiguration>
```
Do you have any suggestions about this? 

Thanks.

**Comment by mattbisme** [comment]
I had the same issue. I am also using the US East server, but I had to use `http://BUCKET.s3.us-east-1.amazonaws.com` for the `BUCKET_URL`. Two important notes:

1. I had to use HTTP, not HTTPS (I suspect additional configuration is needed for HTTPS).
2. The US East does not follow the same naming convention as the other regions. That's why `s3.REGION` is needed instead of `s3-REGION`. [AWS regions can be found here](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region).

After that, empty your cache and refresh. It may also be worth noting that I had to use `S3BL_IGNORE_PATH = true` to get the navigation breadcrumbs to work properly.
